javascript/ql/src/Security/CWE-730/UnclosedStream.qhelp

```diff
@@ -1,22 +1,69 @@
 <!DOCTYPE qhelp PUBLIC
-  "-//Semmle//qhelp//EN"
-  "qhelp.dtd">
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
 <qhelp>
 
-	<overview>
+<overview>
 
-	</overview>
+	<p>
 
-	<recommendation>
+		Callback-based input streams generate calls until they are empty or
+		closed explicitly, and they will take up system resources until
+		that point in time.
```

Comment:
If we rephrase this in terms of events, we can avoid inventing non-standard terms:

```suggestion
		Input streams in Node.js emit events until they are empty or
		closed explicitly, and they will take up system resources until
		that point in time.
```